packages/core/src/Stores/notification-store.js

```diff
@@ -1474,4 +1475,21 @@ export default class NotificationStore extends BaseStore {
             platform: 'Account',
         });
     };
+
+    showAccountSwitchToRealNotification = (loginid, currency) => {
+        const regulation = loginid?.startsWith('CR') ? localize('non-EU') : localize('EU');
```

Comment:
This eu, non_eu is just applicable for low_risk clients, what is expected for others?
clarify this with design team

Reply:
I checked it with login id, so even if the use High risk countries or EU countries. It will show correctly because of the login id

Reply:
But we do not have this classification out of DIEL. It maybe confusing for the user to know what this non_eu means.
let's proceed with this login-id implementation for now.